crowbar_framework/app/helpers/form_helper.rb

```diff
@@ -55,6 +55,31 @@ def fs_for_select(selected)
     )
   end
 
+  def beautify_disk_owner(owner)
+    @owner2label ||= {
+      "Ceph" => t(".disk_role_ceph"),
+      "Cinder" => t(".disk_role_cinder"),
+      "Swift" => t(".disk_role_swift"),
+      "LVM_DRBD" => t(".disk_role_drbd"),
+      "sbd" => t(".disk_role_sbd")
```

Comment:
About RAID - those are set at different time, right? You have to prepare RAID before node installation, and than there's no way (and reason) to change it. But these disk roles are for non-system disks.

About names - I guess you are right, but wouldn't it be better in some other PR?